ReadMe.md

```diff
@@ -50,11 +51,11 @@ Extract the contents of the corresponding zip file to the same folder as Timespi
 * In Seed selection menu, Ctr+C can now be used to copy seed id to the clipboard
 
 # Routing changes
-* The drawbridge is now open by default, and doesn't require you to visit the twins demons
+* The drawbridge is now open by default, and doesn't require you to visit the demon twins
 * Amadeus Laboratory now requires keycard B to open
 * To prevent players from getting stuck in the past when items are still available in the present, the transition room next to the refugee camp can now be used to freely travel back and forth between past and present
 * When you obtain the pyramid keys, a random transition room is unlocked that could open new routing opportunities
-* The temporal gyre is closed, due to its high randomness inside an already randomized timezone, the timezone would be highly unstable
+* The Temporal Gyre's boss rooms are disconnected from its main loop, which instead exits to its entrance. When using the "Gyre Archives" flag, Ifrit can be reached by aquiring Kobo and proceeding to the portrait room in the Library, while Ravenlord can be reached by aquiring Merchant Crow and proceeding to the room in Amadeus' Library containing the Historical Archives
```

Comment:
Are you sure its called Amadeus' Library? i though it was someting like labetory?